SoD_SP_Public/Next Version/SoD_SP.ini

```diff
@@ -325,9 +325,9 @@ FastBattleBugs = 1
 
 # Naga Banks provide Sulfur instead of Gems. Requires New Game to take effect.
 # Сера в Нага Банках вместо драгоценных камней. Изменения вступят в силу только в новой игре.
-# Naga Banks stellen Schwefel statt Edelsteins zur Verfügung. Neus Spiel wird gebraucht um diese Änderung wirksam zu werden. 
+# Naga Banks stellen Schwefel statt Edelsteins zur Verfügung. Neus Spiel wird gebraucht um diese Änderung wirksam zu werden.
 # Τα Naga banks παρέχουν θειάφι αντί πολύτιμων λίθων. Απαιτείται νέο παιχνίδι για να ισχύσει η αλλαγή.
-NagaBank = 0
+NagaBank = 1
```

Comment:
The default ini options should not be modified, just the comments explaining what they do.